### PR TITLE
[FEATURE] Introduce PAGEVIEW template

### PR DESCRIPTION
**Pull request by linawolf**
Depends on #14

### CODE REVIEW

File: Configuration/TypoScript/Setup/Part1PageviewSection.typoscript [hunk 1]
```diff
@@ -0,0 +1,8 @@
+// Part 1: PAGEVIEW section
+page.10 = PAGEVIEW
+page.10 {
+  paths {
+    0 = EXT:site_package/Resources/Private/Templates/
+    10 = {$page.template.path}
```
Comment (Alagts) [inline Configuration/TypoScript/Setup/Part1PageviewSection.typoscript:6]:
```suggestion
    
```

**Review by Alagts — CHANGES_REQUESTED** [review]
I would suggest these changes or grant access to let me push my changes: 
- delete 10 = {$page.template.path} in file Configuration/TypoScript/Setup/Part1PageviewSection.typoscript
- delete complete file Configuration/Sets/SitePackage/constants.typoscript
- delete complete file ext_emconf.php since not needed anymore since v12
- delete complete file ext_localconf.php is not needed because we outsourced page.tsconfig include into config/sites/site_package/page.tsconfig next to the config.yaml file one layer above the site_package.

File: Configuration/TypoScript/Setup/Part1PageviewSection.typoscript (hunk 1 above)
Comment (linawolf) [reply Configuration/TypoScript/Setup/Part1PageviewSection.typoscript:6]: Why would you delete that?

**Comment by linawolf** [comment]
ext_emconf.php is still needed if you use a non-compoer legacy installation. Can you make a separate PR that deletes the ext_localconf.php? The change is not related.
I think defining a template path in the constants to easaly override the templates is considered best practice?

File: Configuration/TypoScript/Setup/Part1PageviewSection.typoscript (hunk 1 above)
Comment (linawolf) [reply Configuration/TypoScript/Setup/Part1PageviewSection.typoscript:6]: This is considered as a common best practice. This way the path can be easily overriden just by setting a constant. Many extensions and sitepackages do this by default

**Comment by linawolf** [comment]
Continued in https://github.com/TYPO3-Documentation/TYPO3CMS-Tutorial-SitePackage-Code/pull/20

**Pull request closed without merging**